site-src/guides/http-request-mirroring.md

```diff
@@ -0,0 +1,22 @@
+# HTTP request mirroring
```

Comment:
Would be able to put a notice at the top of this file that Request Mirroring currently has Extended support and requires the `HTTPRouteRequestMirror` to be supported on the implementation?

Maybe something like this:
```suggestion
# HTTP request mirroring

??? example "Extended Support Feature"

    As of v1.0.0, the Request Mirroring feature is an Extended feature, and requires implementations to support
    the `HTTPRouteRequestMirror` feature.
```

Reply:
Hi Nick thanks for the comment!
Yes I have tested that this works on istio and it worked. Let me know if you would like the result to the PR description!

Reply:
If you use the same formatting I did, it will appear as a neat expandable headline, like at the top of https://gateway-api.sigs.k8s.io/guides/backend-protocol/ . Not a blocker though.